src/toil_scripts/lib/files.py

```diff
@@ -35,6 +35,13 @@ def move_files(file_paths, output_dir):
         shutil.move(file_path, dest)
 
 
+def move_file_job(job, name, file_id, output_dir):
+    """Job version of `move_files` for a single file"""
```

Comment:
`move_files` operates on files that have accessible file paths.  I needed a job version of that function so I could pipe the FileStoreID, returned as a promise, to `move_file_job`. 

I asked Hannes if the one-liner docstring was OK, given that `move_file` is defined right above it and the `job` versions of the URL functions follow the same convention.  I'll go ahead and change it since it you requested.